docs/user/masternodes/understanding.rst

```diff
@@ -81,6 +81,25 @@ real-time payment rates, and `this site
 <http://178.254.23.111/~pub/Dash/Dash_Info.html>`_ for various real-time
 statistics on the masternode network.
 
+High-Performance Masternodes (HPMNs)
```

Comment:
Currently the right-side navigation is weird due to how the heading levels are structured. Let's add a new heading called "Overview" just below the "Understanding Masternodes" one at the top of the document. That heading should be underlined with "=" characters. This should fix the right-side nav issue we introduced.